src/main/java/core/basesyntax/strategy/OperationServiceStrategy.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.service.OperationService;
+import core.basesyntax.service.impl.AddOperationService;
+import core.basesyntax.service.impl.SubtractOperationService;
+
+public class OperationServiceStrategy {

```

Comment:
let's avoid using 'service' here as well. Consider also creating an interface for this class